Xamarin.Forms.Core/ListView.cs

```diff
@@ -210,7 +212,8 @@ public SeparatorVisibility SeparatorVisibility
 			set { SetValue(SeparatorVisibilityProperty, value); }
 		}
 
-		internal ListViewCachingStrategy CachingStrategy { get; private set; }
+		[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
IFooController interfaces are there for when customers need to actually use the methods for implementing renderers. Simply hidden from intellisense is there for when they are not. That is the distinction we need to draw with Controller vs EditorBrowsable